### DISCUSSION TITLE
Dragon's Palace

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1295] [Maps and Content Showcase]
Dragon's Palace(Chinese: 龙宫) is a map based on the Meridian Gate (also called Wumen, Chinese: 午门) in the Forbidden City. This map is suitable for all modes except racing. Cooperation is the key.

**Comment by ghost** [comment]
[Dragon.zip](https://github.com/redeclipse/base/files/6996698/Dragon.zip)

**Comment by ghost** [comment]
![20210817101423](https://user-images.githubusercontent.com/87136890/129656691-c623824f-ea57-48fa-85d6-b471002eda63.png)
![20210817101528](https://user-images.githubusercontent.com/87136890/129656700-5dae9f4e-7f63-440d-97d6-4c804ef6ce10.png)

**Comment by ghost** [comment]
The real Wumen: 
![21680125_155506006000_2](https://user-images.githubusercontent.com/87136890/129658903-8c678594-a77c-4d11-9fb3-5ea5bcd16304.jpg)

**Comment by qreeves** [comment]
While I appreciate the geometry must have taken some time to accomplish, the map could use improvement in several areas:

- The map is too blocky and unnatural looking. It needs more variation and less sharp edges.
- The texturing is very uniform and bland. Try using some different textures that don't all blend into one colour and make use of blendmaps/decals. The hex tiles and concrete pretty much blends in with the bricks, making it difficult to tell wall from floor. The hex tile and stippled metal looks out of place in this setting as well.
- The lighting in both variants has issues. The normal variant is too uniform, and the alternate variant has way too many light sources in a very dark setting. While dark may seem cool and edgy, it makes for very poor gameplay. Too many lights will lead to poor performance in the Tesseract engine.
- Mapmodels like the sci-fi lights seem out of place in what is supposed to be a fairly ancient looking setting.
- The skybox could be more interesting, try `cloudlayer` or `cloudbox` and maybe some `atmo`.

**Comment by cloudziq** [comment]
for sky and environment lighting you can try the REEM, available here: https://github.com/redeclipse/base/discussions/1273

**Comment by ghost** [comment]
Thank you for your useful and professional suggestions. This is my last map, everyone can improve it. I had this map in my dream. Everyone can help me if you think it is beautiful.

**Comment by ghost** [comment]
I'm also grateful for the kindness of dziq.

**Comment by dc-redeclipse** [comment]
the map sounds like a place for me lol 🐉
